server/utils.py

```diff
@@ -253,3 +254,23 @@ def generate_number_table(num):
     Used in models.Assignment.mysql_course_submissions_query
     """
     return ' UNION '.join('SELECT {} as pos'.format(i) for i in range(1, num + 1))
+
+def upload_file(file, container=None, name=None, prefix=None, overwrite=False):
+    """ A utility function to get around having to specify
+    what file extensions are allowed.
+    """
+    extension = get_file_extension(file)
```

Comment:
I didn't want to hardcode all of the files we will accept and have to redeploy every time we change the filetype so I added this code in here. 

I could specify it as a giant list in the constants or settings - but I think this is better.